lib/reline/ansi.rb

```diff
@@ -154,11 +153,14 @@ def self.with_raw_input
   end
 
   @@buf = []
-  def self.inner_getc
+  # a single count is 0.1 sec
```

Comment:
I think `1 count is 0.1 sec` and using `count = timeout_millisecond / 100` in `reline.rb` is hard to understand.
How about this? `minimum_timestep=100ms` will be encapsulated inside inner_getc
```ruby
def self.inner_getc(timeout_second) # or millisecond. I think second is more frequently used in ruby.
  foo
  until c = bar{baz(0.1)&&getbyte)
     timeout_second -= 0.1
     return nil if timeout_second <= 0
     foo
  end
  bar
end
```

Reply:
Yeah I think using `timeout_second` is better. I've updated it.